### DISCUSSION TITLE
Person Folder

### DISCUSSION THREAD

**Discussion by gringojack** [mytourbook/mytourbook#1029] [General]
In Preferences -> Persons, what is "Person Folder" for?
Thank you

**Comment by Thompson-ongithub** [comment]
you can use MTB for more than one person... hence in the Person Folder you find all the people you created within MTB

**Comment by gringojack** [comment]
Just tested the the person folder option. How can I delete the person I just added?

**Comment by wolfgang-ch** [comment]
There is currently no possibility to delete a person
